apps/bajour/utils/shouldDisplayBaselBriefing.ts

```diff
@@ -0,0 +1,12 @@
+const shouldDisplayBaselBriefing = (): boolean => {
+  const now = new Date()
+
+  // Swiss time is UTC+1 or UTC+2 (when daylight saving time is in effect)
+  const offset = now.getTimezoneOffset() === -120 ? 2 : 1 // -120 for CEST (daylight saving), otherwise it's CET
```

Comment:
Does this matter? Most of the users will have CET/CEST set, this would for example lead to a bug when someone from UTC+3 would view the page.

I think it's fine to just check the time.